examples.md

```diff
@@ -0,0 +1,105 @@
+# Mano Simulator Examples
+Here are some simple examples you can do with Mano simulator. You can learn how to write a mano program by taking examples from these codes.
+
+### Change RESULT variable to !X
```

Comment:
Include backticks around all variables in text outside of code blocks (e.g. `RESULT` and `X` instead of RESULT and X)